src/Nox.Ui.Blazor.Lib/Components/Generic/Actions.razor

```diff
@@ -0,0 +1,20 @@
+﻿@using MudBlazor
+
+@namespace Nox.Ui.Blazor.Lib.Components.Generic
+
+@inherits ComponentBase
+
+<MudButton OnClick="OnAddClicked"
+           Variant="Variant.Filled"
+           StartIcon="@Icons.Material.Filled.Add"
+           Color="Color.Info">ADD</MudButton>
```

Comment:
should be localized?

Reply:
have localized all the existing buttons on ui